python_deltabot_plugin/echo_command.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from deltabot.hookspec import deltabot_hookimpl
+
+version = '0.5'
+
+@deltabot_hookimpl
+def deltabot_init(bot):
+    bot.commands.register(name="/echo", func=process_command_echo)
+
+
+def process_command_echo(command, replies):
+    """ Echoes back received message.
+    To use it you can simply send a message starting with
+    the command '/echo'. Example: `/echo hello world`
+    """
+    message = command.message
+    contact = message.get_sender_contact()
+    sender = 'From: {} <{}>'.format(contact.display_name, contact.addr)
```

Comment:
```suggestion
    sender = 'From: {} <{}>'.format(contact.name, contact.addr)
```
IIRC `contact.display_name` is deprecated

Reply:
get_first_name was the thing that was deprecated you might have confused the two.